src/components/apiTokens/GenerateActionButton.tsx

```diff
@@ -34,16 +21,15 @@ function GenerateActionButton({
                     Cancel
                 </ButtonWithLoader>
                 <ButtonWithLoader
-                    rootClassName="flex cta h-36"
-                    onClick={() => onSave()}
+                    rootClassName= {`flex cta h-36 ${regenerateButton ? 'second' : 'mr-12 first'}`}
```

Comment:
can we rename first &b second like `order-first`  or more specific

Reply:
done